src/Cxx/Utilities/PCADemo.cxx

```diff
@@ -146,9 +146,11 @@ int main(int, char*[])
   double p0[2];
   p0[0] = -100*evec1->GetValue(0);
```

Comment:
Looks like p0[2] should be p0[2]. Likewise for p1.

Reply:
Indeed. Fixed.